releases/release-1.22/release-notes/maps/pr-101171-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 101171
+releasenote:
+  text: ' The new flag `--chunk-size=SIZE` has been promoted to beta, and enabled
+    by default. This flag may be used to alter the number of items or disable this
+    feature when `0` is passed.'
```

This looks like SIG CLI, which has different promotion mechanism. I would ping `#sig-cli` to clarify "beta, and enabled by default" (the enabled-by-default policy is usually server-side feature gate which can be enforced).

```suggestion
  text: ' The new flag `--chunk-size=SIZE` has been added to `kubectl describe`.
    This flag may be used to alter the number of items or disable this feature when
    `0` is passed.'
```